ecpy_hqc_legacy/manifest.enaml

```diff
@@ -263,7 +298,8 @@ enamldef HqcLegacyManifest(PluginManifest):
                 Task:
                     task = 'apply_mag_field_task:ApplyMagFieldTask'
                     view = 'views.apply_mag_field_view:ApplyMagFieldView'
-                    instruments = ['ecpy_hqc_legacy.Legacy.IPS12010']
+                    instruments = ['ecpy_hqc_legacy.Legacy.IPS12010',
+                                   'ecpy_hqc_legacy.Legacy.CS4']
```

Comment:
They share the same driver (CS4); I did add 4G as a separate Driver extension in the manifest